code/game/objects/structures/musician.dm

```diff
@@ -321,6 +327,10 @@
 	else
 		return TRUE
 
+/datum/song/handheld/toggle_playing(user, new_play_state)
+	. = ..()
+	var/obj/item/instrument/instrument = instrumentObj

```

Comment:
I don't really see the need relegating to subtype, this is just instrument matter. Especially since you don't se that subtype anywhere.